editor/editor_node.cpp

```diff
@@ -3946,6 +3946,7 @@ bool EditorNode::is_addon_plugin_enabled(const String &p_addon) const {
 void EditorNode::_remove_edited_scene(bool p_change_tab) {
 	// When scene gets closed no node is edited anymore, so make sure the editors are notified before nodes are freed.
 	hide_unused_editors(SceneTreeDock::get_singleton());
+	hide_unused_editors(NodeDock::get_singleton());
```

Comment:
Pretty sure NodeDock is never editing owner of anything, so this line is wrong.
I tested that it's unnecessary for fixing the bug.